### DISCUSSION TITLE
a few questions about the premine & default staked amount of PoS

### DISCUSSION THREAD

**Discussion by eddieran** [0xPolygon/polygon-edge#628] [Q&A]
### Description

when trying generate the genesis json, i setup some premine amount and also specify the ibft-validator like
```
    --premine 0xD5655907aa7A8F9d3aD10c2257d9809D35A28897:1000000000000000000000 \
    --premine 0x08E33B003271b3790a8922022F870264Da74ba72:1000000000000000000000 \
    --ibft-validator 0xD5655907aa7A8F9d3aD10c2257d9809D35A28897 \
    --ibft-validator 0x08E33B003271b3790a8922022F870264Da74ba72 \
```

but also, while i go through the process of generating genesis, there would be default staked balance value
![image](https://user-images.githubusercontent.com/8403607/177907569-ba827eb2-1d8a-423d-a29a-e55e7b04ba76.png)

i want to know 
1. How does this work, is a default staked amount granted to each validator at startup? If so, why is it designed this way?
2. What if I give my validator A address and premine 1000 ETH, but this default value will still use the default staked amount? 
3. Is there any way to avoid the default staked amount? Or any suggestions for starting a PoS chain?

**Comment by laviniat1996** [comment]
Hello @SureEdding,
Thank you for your questions!

I will answer each of them in the order they were asked:
1. Yes. DefaultStakedBalance is deposited in the Staking contract for each validator, while the premine balance is given to the accounts.
2. Yes. Currently the premine balance doesn’t affect  thegenesis stake amount.
3. Right now there is no way to avoid it, but you can make the stake balance configurable.
